### PR TITLE
Resolve some components of #228

### PR DESCRIPTION
**Pull request by bwohlberg**
Resolve some components (as indicated in issue) of #228.

### CODE REVIEW

File: examples/scripts/README.rst [hunk 1]
```diff
@@ -10,74 +10,74 @@ Computed Tomography
...
+   `deconv_ppp_bm4d_admm.py <deconv_ppp_bm4d_admm.py>`_
+      PPP (with BM4D) Volume Deconvolution
 
 
 Sparse Coding
 ^^^^^^^^^^^^^
 
    `sparsecode_admm.py <sparsecode_admm.py>`_
-      Non-negative Basis Pursuit DeNoising (ADMM)
+      Non-Negative Basis Pursuit DeNoising (ADMM)
```
Comment (tbalke) [inline examples/scripts/README.rst:57]: Did we find a consensus on how we handle capitalization?
I find it odd to read `COde` or `DeNoising`
@Michael-T-McCann

File: examples/scripts/ct_abel_tv_admm.py [hunk 2]
```diff
@@ -5,13 +5,22 @@
...
 
 r"""
-Regularized Abel Inversion
-==========================
+TV Regularized Abel Inversion
+=============================
 
-This example demonstrates a TV-regularized Abel inversion using
-an Abel projector based on PyAbel :cite:`pyabel-2022`
+This example demonstrates a TV-regularized Abel inversion by solving the
```
Comment (tbalke) [inline examples/scripts/ct_abel_tv_admm.py:11]: `TV-regularized` or `TV regularized`?

File: examples/scripts/README.rst (hunk 1 above)
Comment (bwohlberg) [reply examples/scripts/README.rst:57]: This is non-standard capitalization, but it's the one originally used to justify the BPDN acronym. I'm open to alternatives if anyone finds this version truly offensive.

File: examples/scripts/ct_astra_tv_admm.py [hunk 3]
```diff
@@ -5,16 +5,14 @@
 # with the package.
 
 r"""
-Few-View CT (ADMM w/ Total Variation)
-=====================================
+TV Regularized Few-View CT Reconstruction
```
Comment (tbalke) [inline examples/scripts/ct_astra_tv_admm.py:8]: Is few-view the same as sparse-view? I never heard it before.

File: examples/scripts/ct_svmbir_ppp_bm3d_admm_prox.py [hunk 4]
```diff
@@ -5,13 +5,12 @@
...
 
-This example demonstrates the use of class
-[admm.ADMM](../_autosummary/scico.optimize.rst#scico.optimize.ADMM) to
-solve a tomographic reconstruction problem using the Plug-and-Play Priors
-framework :cite:`venkatakrishnan-2013-plugandplay2`, using BM3D
+This example demonstrates solution of a tomographic reconstruction
+problem using the Plug-and-Play Priors framework
+:cite:`venkatakrishnan-2013-plugandplay2`, using BM3D
 :cite:`dabov-2008-image` as a denoiser and SVMBIR :cite:`svmbir-2020` for
 tomographic projection.
```
Comment (tbalke) [inline examples/scripts/ct_svmbir_ppp_bm3d_admm_prox.py:16]: change `version` to `example` here as well?

File: examples/scripts/ct_svmbir_ppp_bm3d_admm_prox.py [hunk 5]
```diff
@@ -5,13 +5,12 @@
 # with the package.
 
 """
-CT Reconstruction (ADMM Plug-and-Play Priors w/ BM3D, SVMBIR+Prox)
-==================================================================
+PPP (with BM3D) CT Reconstruction (ADMM with SVMBIR Fast Prox)
```
Comment (tbalke) [inline examples/scripts/ct_svmbir_ppp_bm3d_admm_prox.py:8]: Maybe call it `Fast SVMBIR Prox`?

**Review by tbalke — APPROVED** [review]
Approve upon addressing a few minor comments.

File: examples/scripts/ct_abel_tv_admm.py (hunk 2 above)
Comment (bwohlberg) [reply examples/scripts/ct_abel_tv_admm.py:11]: I believe `TV-regularized` is correct.

File: examples/scripts/ct_astra_tv_admm.py (hunk 3 above)
Comment (bwohlberg) [reply examples/scripts/ct_astra_tv_admm.py:8]: Changed `few-view` to `sparse-view`.

File: examples/scripts/ct_svmbir_ppp_bm3d_admm_prox.py (hunk 4 above)
Comment (bwohlberg) [reply examples/scripts/ct_svmbir_ppp_bm3d_admm_prox.py:16]: Some rephrasing in both versions.

File: examples/scripts/ct_svmbir_ppp_bm3d_admm_prox.py (hunk 5 above)
Comment (bwohlberg) [reply examples/scripts/ct_svmbir_ppp_bm3d_admm_prox.py:8]: Done.

File: examples/scripts/ct_abel_tv_admm.py (hunk 2 above)
Comment (bwohlberg) [reply examples/scripts/ct_abel_tv_admm.py:11]: Not anymore!

**Pull request merged**